unit-tests/func/frame-filter/test-l500-frame-filter.py

```diff
@@ -0,0 +1,60 @@
+import pyrealsense2 as rs, test
+import time
+
+# The L515 device opens the IR steam with the depth stream even if the user did not asked for it (for improving the depth quality), 
```

Comment:
... IR `stream`... did not `ask` for it ...
Line 6:
... is `divided` ... 2 `tests:`

Reply:
👍